gluten-uniffle/velox/src/main/java/org/apache/spark/shuffle/gluten/uniffle/UniffleShuffleManager.java

```diff
@@ -57,6 +57,8 @@ public <K, V> ShuffleWriter<K, V> getWriter(
       } else {
         writeMetrics = context.taskMetrics().shuffleWriteMetrics();
       }
+      sparkConf.set(
+          RssSparkConfig.SPARK_RSS_CONFIG_PREFIX + RssSparkConfig.RSS_ROW_BASED.key(), "false");
```

Comment:
Is it possible to create a fallbacked shuffle writer and then create a non-fallbacked shuffle writer?

Reply:
For example, there are two stages running on the same executor, one shuffle without fallback and one with fallback

Reply:
It's necessary to change initialization of `RssShuffleWriter` in uniffle to inform `WriteBufferManager` use row serialization rather than columnar serialization